client/src/App.css

```diff
@@ -0,0 +1,106 @@
+:root {
+  --sidebar-width: 100px;
+  --sidebar-bg-colour: white;
+}
+
+a {
+  text-decoration: none;
+  font: bold;
+  color: black;
+}
+
+/*.wrapper {*/
+/*  text-align: center;*/
+/*  display: flex;*/
+/*  flex-wrap: wrap;*/
+/*  right: 0;*/
+/*}*/
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.nav-li {
+  padding: 0.5rem;
+  right: 0;
+  text-decoration: none;
+}
+
+/*.menu-toggle {*/
+/*  display: initial;*/
+/*  margin: 0.5rem;*/
+/*  position: absolute;*/
+/*  right: 0;*/
+/*}*/
+
+/*.bar1, .bar2, .bar3 {*/
+/*  width: 35px;*/
+/*  height: 5px;*/
+/*  background-color: #333;*/
+/*  margin: 6px 0;*/
+/*  transition: 0.5s;*/
+/*}*/
```

Comment:
At some point in the future remove commented out code. It should probably be a final cleaning step when you are done with the main features